code/game/machinery/emancipation.dm

```diff
@@ -0,0 +1,58 @@
+/obj/machinery/emancipation_grill
+	name = "emancipation grill"
+	desc = "Liberator of luggage. Larcenist of belongings."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "emancipation_grill"
+	density = 0
+	anchored = 1
+	use_power = 1
+	idle_power_usage = 75
+	active_power_usage = 750
+	var/list/obj_whitelist = list() //Things that are okay to go through. Frazzle everything else.
+	var/list/obj_blacklist = list() //Things that aren't okay to go through. Don't frazzle everything else.
+
+/obj/machinery/emancipation_grill/New()
+	..()
+	update_icon()
+
+/obj/machinery/emancipation_grill/power_change()
+	. = ..()
+	update_icon()
+
+/obj/machinery/emancipation_grill/update_icon()
+	if(stat & (BROKEN|NOPOWER))
+		icon_state = initial(icon_state)
+	else
+		icon_state = "[initial(icon_state)]_on"
+
+/obj/machinery/emancipation_grill/proc/emancipate(atom/movable/user)
+	if(!user)
+		return
+	if(stat & (BROKEN|NOPOWER))
+		return
+	use_power = 2
+	if(isobj(user)) //Let's only vaporize objects
```

Comment:
Is there any reason for this because this just seems arbitrary.

Reply:
I think its from the code I posted in the issue, I wanted it to use power so when you complete the puzzle that I had in mind you turn off the equipment power and can walk out with prize

Reply:
That's... completely unrelated?

Reply:
Thats the power code no?

Reply:
No I'm talking about the line that only allows objs to be emancipated. Not the power code.

Line comments always go about the last line you see.

Reply:
It's so you didn't have to add every sort of mob to an emancipation grill whitelist.

Object emancipation only

Reply:
That's alright then.